director/director_api.go

```diff
@@ -33,3 +47,68 @@ func ListNamespacesFromOrigins() []NamespaceAd {
 	}
 	return namespaces
 }
+
+func LoadDirectorPublicKey() (*jwk.Key, error) {
+	discoveryUrlStr := param.Federation_DirectorUrl.GetString()
```

Comment:
Just to make greping the repo a little easier, can you modify `discoveryUrl` stuff to include some reference to `director`? We use `discoveryUrl` elsewhere to indicate the federation's discovery url